templates/download/server/arm.html

```diff
@@ -121,10 +154,14 @@ <h3 class="p-heading--5">Installation guides</h3>
           <h3 class="p-heading--5">Professional support for Ubuntu</h3>
         </div>
         <div class="col-6 col-medium-3">
-          <p>Get professional support for Ubuntu from Canonical. We help organisations around the world to manage their
-            Ubuntu cloud, server and desktop deployments.</p>
+          <p>
+            Get professional support for Ubuntu from Canonical. We help organisations around the world to manage their
+            Ubuntu cloud, server and desktop deployments.
```

Comment:
This break doesn't look right?

Reply:
Strange, I don't see the break in the conversation tab.

Reply:
Don't see an issue here, on the page it shows correctly. I think linter applied 120 max symbols per line rule here.